### ISSUE TITLE
Compile regression model for TPU problem with op: div_no_nan

### ISSUE THREAD

**Issue by adr-arroyo** [google-coral/edgetpu#56]
Hello,

I have been trying (unsucessfully) to create and compile a model used for regression (final activation is _linear_ instead of f.e. _softmax_). After several attemps with TF versions _1.15_ and _2.1_ I have seen that the problem is the operation `div_no_nan` which is performed within the scope of the loss function, as shown in this image from Tensorboard: 

![tensorboard_tpu_incompatibility](https://user-images.githubusercontent.com/11291505/74138357-c30f9000-4bf1-11ea-99b6-ba028986e071.png)

I have tried to use different types of Optimizers, such as _RMSProp_ or _Adagrad_, but the problem remains the same: edge_tpu_compiler throwing an _unrecognizable_ compiler error, which is caused by this operation (hopefully only this one).

The code I am using to create the model is:

```
def get_model(self, shape, num_forecasts, inner_activation='relu'):
        model = tf.keras.models.Sequential()
        model.add(tf.keras.layers.Dense(units=int(64), input_shape=shape, activation=inner_activation))
        model.add(tf.keras.layers.Dense(units=int(64), activation=inner_activation))
        model.add(tf.keras.layers.Flatten())
        model.add(tf.keras.layers.Dense(units=num_forecasts, activation="linear"))

        # opt = tf.keras.optimizers.Adam(lr=0.01)

        model.compile(optimizer="adam",loss="mae", metrics=['acc'])
        model.summary()
        return model
```

The ops I use to convert the model to TPU are:

```
        if tf.__version__.split(".")[0] == '1':
            converter.target_ops = [tf.lite.OpsSet.TFLITE_BUILTINS_INT8]        
        else:
            converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS_INT8]
        converter.representative_dataset = self.representative_dataset_gen
        converter.inference_input_type = tf.uint8
        converter.inference_output_type = tf.uint8
        converter.optimizations = [tf.lite.Optimize.DEFAULT]
```

My question is then, is there any way/combination of optimizer and loss function to prevent this error (and any TPU incompatibility) for regression instead of classification?

In addition, I provide here the model (in keras format) and the TFLite version (might not be the exact same model shown in the image or the code but the problem is the same).

[models.zip](https://github.com/google-coral/edgetpu/files/4180062/models.zip)

Regards, 
Adrian

**Comment by Namburger** [comment]
Hi Adrian, 
can you check your compiler version? It appears that you've hit a bug on an older compiler that is already fixed. The current release is: 
```
Edge TPU Compiler version 2.0.291256449
```
I've attached the compiled version :)

**Comment by adr-arroyo** [comment]
Hello,

Can't believe that the problem was on the compiler version :S after two weeks of trying a lot of model architectures. Thanks @Namburger 

I can see that the compiled model only has 1 layer running on the TPU, I guess the operations are still no compatible.

Any suggestions on fixing this?

Regards :)
